CHANGELOG.rst

```diff
@@ -27,6 +27,11 @@ Changed
 
   Contributed by @punkrokk
 
+* Added an enhancement where ST2api.log no longer reports the entire traceback when trying to get a datastore value
+  that does not exist. It now reports a simplified log for cleaner reading. Addresses and Fixes #4979. (improvement) #4981
+
+Contributed by Justin Sostre (@saucetray)
```

Comment:
Please fix the indentation here. It will format this line as separated to the changelog list item.